ceph/ceph.go

```diff
@@ -112,63 +128,125 @@ func (ceph *Ceph) Init(config map[string]ctypes.ConfigValue) error {
 	return nil
 }
 
-// GetCephDaemonMetrics executes "ceph --admin-daemon perf dump" command for defined daemon-socket and returns its metrics
-func (ceph *Ceph) GetCephDaemonMetrics(mts []plugin.MetricType, daemon string) ([]plugin.MetricType, error) {
-	out, err := cmd.perfDump(filepath.Join(ceph.path, "ceph"), "--admin-daemon", filepath.Join(ceph.socket.path, daemon),
-		"perf", "dump")
+// GetCephDaemonMetrics executes "ceph --admin-daemon perf dump" command for defined daemon-socket and returns metrics matching with mts
+// If provided metric list is nil, function returns all available Ceph metrics using "ceph --admin-daemon perf schema" command
+// Delta metrics (sum / avgcount) are calculated automatically
+func (ceph *Ceph) GetCephDaemonMetrics(mts []plugin.Metric, daemon Daemon) ([]plugin.Metric, error) {
 	timestamp := time.Now()
```

Comment:
It seems that this variable is not needed, you can call `time.Now()` directly in line 173.